common/resource.go

```diff
@@ -365,7 +365,7 @@ func genericDatabricksData[T, P, C any](
 	hasOther bool) Resource {
 	var dummy T
 	var other P
-	otherFields := StructToSchema(other, NoCustomize)
+	otherFields := StructToSchema(other, nil)
```

Comment:
do we still need it? 

@edwardfeng-db - can you check?

Reply:
Yeah this looks good